doc/source/whatsnew/v1.2.0.rst

```diff
@@ -314,6 +314,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrameGroupby.tshift` failing to raise ``ValueError`` when a frequency cannot be inferred for the index of a group (:issue:`35937`)
 - Bug in :meth:`DataFrame.groupby` does not always maintain column index name for ``any``, ``all``, ``bfill``, ``ffill``, ``shift`` (:issue:`29764`)
 - Bug in :meth:`DataFrameGroupBy.apply` raising error with ``np.nan`` group(s) when ``dropna=False`` (:issue:`35889`)
+- Bug when combining methods :meth:`DataFrame.groupby` with :meth:`DataFrame.resample` and restricting to `Series` or using `agg` did miscalculate the aggregation (:issue:`27343`, :issue:`33548`, :issue:`35275`).
```

Comment:
can you elabortate what 'miscalculate' means here

Reply:
Two things can happen here right now:
- If the Index of the Input df has any index except an RangeIndex starting at 0, it crashes
- If the index is a RangeIndex, the obj.index keeps the previous index labels. These labels are used to extract Index components of self._grouper. But self._grouper is resorted most of the times in resample. So we select random index components which may or may not be correct. Most of the times this results in values getting assigned to different dates as they were in the input df (for example #33548)